.travis.yml

```diff
@@ -16,8 +16,10 @@ jdk:
 #  srcclr: true 
 addons:
   sonarcloud:
-    organization: "apache" 
+    organization: apache
+
 script:
   # other script steps might be done before running the actual analysis
-  - sonar-scanner
-#  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar
```

Comment:
This line is actually the one which should work out of the box, with JaCoCo being activated to get coverage reports.